content-repo/extra-docs/packs/integrations-and-incident-health-check.md

```diff
@@ -121,6 +121,8 @@ You can review the playbook inputs and edit them by clicking the 'Playbook Trigg
 | AutoCloseInvestigation | This input determines if the investigation should close automatically after rerunning the scripts.|
  ---
 
+## Notes
+1. To prevent exceeding quota limits, configured instances in which the *Do not use by default* checkbox was selected, will not be tested by this pack.
```

Comment:
```suggestion
1. Integration instances in that has the *Do not use by default* checkbox was selected, are not supported by this pack. This is due to XSOAR platform limitations and n order to prevent exceeding quota limits
```